system-config/vagrant-reload.rb

```diff
@@ -0,0 +1,77 @@
+# Copyright (c) 2013 Aidan Nagorcka-Smith
+#
+# MIT License
```

Comment:
Perhaps it would be good to update the README.md to say that code in this repo is licensed as MPL-2.0 unless otherwise stated (this being such a statement). Perhaps @jeremiah can help out with exactly how to work with this when incorporating 3rd party code.